src/app/gobravely/index.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable prettier/prettier */
```

Comment:
I don't like too much disabling the linter but I think I understand why you did it. Would it be too much of a problem to fix it? `npm run lint -- --fix`

Reply:
hmm, yeah, I disabled it just to make it more graphic on the code. but there is no problem to remove that  now, It will make the easteregg more of a suprisse now haha